data/abixen-platform/misuses/1/misuse.yml

```diff
@@ -0,0 +1,15 @@
+api:
+- javax.crypto.spec.IvParameterSpec
+violations:
+- insecure/condition/randomization
+description: >
+  First parameter while initializing the IvParameterSpec object was not properly randomized.
+location:
+  file: com/abixen/platform/service/businessintelligence/multivisualisation/domain/model/util/AES128Encoder.java
+  method: "decryptPassword(String)"
+  line: 66
+internal:
+pattern:
```

Comment:
This key is obsolete and can be removed.

Reply:
Just to clarify: You mean the key `pattern`?